stream-chat-android-ui-components/src/main/res/layout/stream_ui_link_attachments_view.xml

```diff
@@ -25,8 +25,9 @@
 
     <FrameLayout
         android:id="@+id/linkPreviewContainer"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        app:layout_constraintDimensionRatio="1.667"
```

Comment:
A ratio of `1.667` means that the preview will be slightly smaller than it previously was, but it seems more appealing and functional to me

Let me know if you think otherwise.